modules/custom/openy_system/openy_system.info.yml

```diff
@@ -0,0 +1,13 @@
+name: OpenY System
+type: module
+description: 'Handles general OpenY configuration for administrators.'
+package: OpenY
+version: VERSION
+core: 8.x
+configure: openy_system.modules_list
+dependencies:
+  # TODO: Show warning message in OpenY components page on openy_demo_webform uninstall
```

Comment:
Add a task to GitHub, please

Reply:
Done - https://github.com/ymcatwincities/openy/issues/1070